tests/libcxx/expected_results.txt

```diff
@@ -412,14 +517,20 @@ std/utilities/meta/meta.trans/meta.trans.other/aligned_storage.pass.cpp FAIL
 std/depr/depr.c.headers/math_h.pass.cpp FAIL
 std/numerics/c.math/cmath.pass.cpp FAIL
 
-# GH-1596: <algorithm>: unqualified calls to _Adl_verify_range incorrectly cause instantiation
-std/algorithms/robust_against_adl.pass.cpp FAIL
+# GH-2358: <filesystem>: path's comparison operators are IF-NDR
+std/input.output/filesystems/class.path/range_concept_conformance.compile.pass.cpp FAIL
+
+# GH-1374: Spaceship CPO wording in [cmp.alg] needs an overhaul
+# (Technically an STL bug until I fix the wording in the working draft to agree.)

```

Comment:
Our comments don't usually speak in the first person singular like this. Not absolutely necessary to change, though.